yt/data_objects/particle_trajectories.py

```diff
@@ -99,9 +99,9 @@ def __init__(
         ):
             dd = ds.all_data()
             newtags = dd[fds["particle_index"]].d.astype("int64")
-            mask = np.in1d(newtags, indices, assume_unique=True)
+            mask = np.isin(newtags, indices, assume_unique=True)
```

Comment:
is the returned shape the same?

Reply:
yes. It matches the shape of the first argument in both cases.